src/argilla/client/feedback/training/schemas.py

```diff
@@ -982,6 +1015,92 @@ def _prepare_for_training_with_trl(
         return ds
 
 
+class TrainingTaskForQuestionAnsweringFormat(BaseModel):
+    """
+    Union[Tuple[str, str, str], List[Tuple[str, str, str]]]
+    """
+
+    format: Union[Tuple[str, str, str], List[Tuple[str, str, str]]]
```

Comment:
The other PR for sentence similarity had this too - do we really need this in a separate class?

Reply:
For me it was the easiest wat to do nested check for the values in List/Tuples dict etc. What do you propose?

Reply:
You're right, it is probably the easiest. I think it's fine to keep it